src/views/Header/HeaderBackButton.js

```diff
@@ -104,9 +104,8 @@ class HeaderBackButton extends React.PureComponent {
     let button = (
       <TouchableItem
         accessible
-        accessibilityComponentType="button"
+        accessibilityRole="button"
         accessibilityLabel={title ? `${title}, back` : 'Go back'}
-        accessibilityTraits="button"
```

Comment:
Can we keep both for backward compatibility?

Reply:
I think yes

Reply:
Let's do it then!